deploy/backplane/acs/01-acs-admins-project.ClusterRole.yaml

```diff
@@ -32,4 +32,48 @@ rules:
   verbs:
   - get
   - list
+  - watch
+# ACS SRE can view routes
+- apiGroups:
+  - route.openshift.io
+  resources:
+  - routes
+  verbs:
+  - get
+  - list
+  - watch
+# ACS SRE can view egress firewalls
+- apiGroups:
+  - k8s.ovn.org
+  resources:
+  - egressfirewalls
+  verbs:
+  - get
+  - list
+  - watch
+# ACS SRE can view roles and rolebindings
+- apiGroups:
+  - rbac.authorization.k8s.io
+  resources:
+  - roles
+  - rolebindings
+  verbs:
+  - get
+  - list
+  - watch
+# ACS SRE can patch persistentvolumeclaims
+- apiGroups:
+  - ""
+  resources:
+  - persistentvolumeclaims
+  verbs:
+  - patch
```

Comment:
Is get/list/watch granted somewhere else?  There is cluster scope permissinos for PV but not PVC.

Reply:
Added view permissions to the aggregate role